examples/dbclient/pokemons/src/main/java/io/helidon/examples/dbclient/pokemons/jdbc/InitializeDb.java

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright (c) 2019, 2020 Oracle and/or its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.helidon.examples.dbclient.pokemons.jdbc;
+
+import java.util.concurrent.CompletionStage;
+import java.util.concurrent.ExecutionException;
+
+import io.helidon.dbclient.DbClient;
+import io.helidon.dbclient.DbExecute;
+
+/**
+ * Initialize JDBC database schema and populate it with sample data.
+ */
+public class InitializeDb {
+
+    /** Pokemon types source file. */
+    private static final String TYPES = "/Types.json";
+    /** Pokemons source file. */
+    private static final String POKEMONS = "/Pokemons.json";
+
+    /**
+     * Initialize JDBC database schema and populate it with sample data.
+     * @param dbClient database client
+     */
+    static void init(DbClient dbClient) {
+        try {
+            if (!PokemonMain.mongo) {
+                initSchema(dbClient);
+            }
+            initData(dbClient);
+        } catch (ExecutionException | InterruptedException ex) {
+            System.out.printf("Could not initialize database: %s", ex.getMessage());
+        }
+    }
+
+    /**
+     * Initializes database schema (tables).
+     *
+     * @param dbClient database client
+     */
+    protected static void initSchema(DbClient dbClient) {
```

Comment:
Why is this protected? Either make the methods public, or package local.

Reply:
It's from the time when part of thew code was in common.